.github/workflows/deploy.yml

```diff
@@ -121,3 +121,9 @@ jobs:
           "
       - name: Import demo course
         run: $SSH "$TUTOR local importdemocourse"
+
+      - name: Import test course & libraries
+        run: |
+            git checkout https://github.com/openedx/openedx-test-course
```

Comment:
This command needs to be wrapped in `$SSH`, such that it is executed on the remote host, and not in the github CI runner.

Reply:
@regisb Of course. Fixed.

I also changed `make import` so that it respects the value of `$TUTOR` instead of hard-coding `tutor`.